admin_guide/persistent_storage_nfs.adoc

```diff
@@ -167,3 +167,8 @@ The export definition above allows arbitrary network clients to mount this
 volume. Exports can be restricted to a range of IP addresses for hosts that will
 access the volume. See `man exports` for more information.
 ====
+ 
+[ADMONITION]
+====
+For OpenShift 3.1 and greater the export values have changed. Please follow this guide to ensure proper security for NFS link:https://docs.openshift.com/enterprise/3.1/install_config/persistent_storage/persistent_storage_nfs.html[NFS setup for OpenShift 3.1]
```

Comment:
Starting in OpenShift Enterprise 3.1, the export values have changed. See the https://docs.openshift.com/enterprise/3.1/install_config/persistent_storage/persistent_storage_nfs.html[OpenShift Enterprise 3.1 documentation] for instructions on ensuring proper security for NFS in 3.1.